test-framework/src/main/java/org/apache/brooklyn/test/framework/TestHttpCallImpl.java

```diff
@@ -50,19 +55,25 @@
      */
     public void start(Collection<? extends Location> locations) {
         String url = null;
+        HttpMethod method = null;
```

Comment:
Minor; feel free to ignore :-)
We only declare `url` above because the variable is needed in the catch block for a nicer error message. I'd declare these variables in-line, rather than outside of the try block, unless we also want to log any of them in the catch error message.